src/emu.c

```diff
@@ -755,15 +871,15 @@ int emu_manager_configure (bool live, int mode) {
     }
 
     if (!(emu->flags & EMU_FLAG_ENABLED)) {
-      emu->flags = 0;
+      emu->flags = 0; // Reset all flags. Must easy to check instead of: `flags & EMU_FLAG_ENABLED`.
```

Comment:
The comment doesn't mean anything. What did you mean?
`Must BE easy...`?

Reply:
Yes!